### PR TITLE
C PLUS PLUS BALS

### PR DESCRIPTION
**Pull request by ongods**
BALS IN EVERY ALUAGE

### CODE REVIEW

**Comment by ongods** [comment]
BALLS

File: balls.cpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// BALLS
+#include <iostream>
+
+void main(){
```
Comment (jesse103) [inline balls.cpp:4]: commit death

Comment (ongods) [reply balls.cpp:4]: NOP

**Comment by ballsballsballsballsballsballsballsball** [comment]
c++ also scares  me

**Comment by sufficients** [comment]
same

**Pull request merged**